datafusion/physical-expr/src/planner.rs

```diff
@@ -106,6 +106,14 @@ pub fn create_physical_expr(
     input_dfschema: &DFSchema,
     execution_props: &ExecutionProps,
 ) -> Result<Arc<dyn PhysicalExpr>> {
+    use datafusion_physical_expr_common::physical_expr::create_physical_expr as create_physical_expr_common;
+
+    // Temporary solution, after all the logic is moved to common, we can remove this function
+    let res = create_physical_expr_common(e, input_dfschema, execution_props);
+    if res.is_ok() {
+        return res;
+    }
+
```

Comment:
For the cases below (like `Expr::Alias`) that were moved, can we please remove them from this list? I think it would make it clearer that that code is now dead (and make it easier to track what remains to port

Reply:
It is not possible yet since `create_physical_expr_common` failed most of the time because many other expression is not yet supported, so we fall back to the original one.

Reply:
> easier to track what remains to port
I think we can just check what is still commented out in `common`